source/frontend/src/features/documents/documentUpload/SelectedDocumentHeader.tsx

```diff
@@ -0,0 +1,139 @@
+import clsx from 'classnames';
+import { FormikProps } from 'formik';
+import truncate from 'lodash/truncate';
+import { useEffect, useMemo } from 'react';
+import { Col, Row } from 'react-bootstrap';
+import { FaCheck, FaTrash } from 'react-icons/fa';
+import styled, { useTheme } from 'styled-components';
+
+import { StyledRemoveIconButton } from '@/components/common/buttons';
+import { Select, SelectOption } from '@/components/common/form';
+import { SectionField } from '@/components/common/Section/SectionField';
+import useIsMounted from '@/hooks/util/useIsMounted';
+import { ApiGen_Concepts_DocumentType } from '@/models/api/generated/ApiGen_Concepts_DocumentType';
+import { exists } from '@/utils';
+import { withNameSpace } from '@/utils/formUtils';
+
+import { BatchUploadFormModel, DocumentUploadFormData } from '../ComposedDocument';
+
+export interface ISelectedDocumentHeaderProps {
+  // props
+  index: number;
+  namespace?: string;
+  className?: string;
+  'data-testId'?: string;
```

Comment:
are single quote needed?

Reply:
Yes, for otherwise the dash in the middle causes issues